agent/src/agent/modules/logger.py

```diff
@@ -5,8 +5,21 @@
 from logging.handlers import RotatingFileHandler
 
 
-def get_logger(name, level=logging.DEBUG, stdout=False):
+levels = {
+    'CRITICAL': logging.CRITICAL,
+    'FATAL': logging.FATAL,
+    'ERROR': logging.ERROR,
+    'WARN': logging.WARNING,
+    'WARNING': logging.WARNING,
+    'INFO': logging.INFO,
+    'DEBUG': logging.DEBUG,
+    'NOTSET': logging.NOTSET,
+}
+
+
+def get_logger(name, level=None, stdout=False):
     logger = logging.getLogger(name)
+    level = level or levels.get(os.environ.get('LOG_LEVEL'), logging.INFO)
```

Comment:
Levels are duplicated with default logging lib. How about this:
`level = level or os.environ.get('LOG_LEVEL', logging.INFO)`

Next function `logger.setLevel(level)` accepts both string and int values

Reply:
What about typoff case?

Reply:
For example, LOG_LEVEL == 'critical'. 
In this case of suggested change, we will have an exception, like: `Unknown level: 'critical'`
In case of double get() we will have INFO level, which is confusing I think

Reply:
Ok, I will use try-except for this case